snorkel/classification/training/trainer.py

```diff
@@ -509,3 +510,66 @@ def _reset_losses(self) -> None:
         """Reset the loss counters."""
         self.running_losses = defaultdict(float)
         self.running_counts = defaultdict(int)
+
+    def save(self, trainer_path: str) -> None:
+        """Save the trainer config to the specified file path in json format.
+
+        Parameters
+        ----------
+        trainer_path :
```

Comment:
nit: no `:` here. see https://github.com/snorkel-team/snorkel/blob/master/snorkel/classification/multitask_classifier.py for example of doc strings